core/lib/via_btc_client/examples/deposit.rs

```diff
@@ -0,0 +1,88 @@
+use std::env;
+
+use anyhow::{Context, Result};
+use bitcoin::{address::NetworkUnchecked, Amount, Txid};
+use tracing::info;
+use via_btc_client::{
+    inscriber::Inscriber,
+    types::{
+        BitcoinAddress, BitcoinNetwork, InscriptionConfig, InscriptionMessage, L1ToL2MessageInput,
+        NodeAuth, Recipient,
+    },
+};
+use zksync_types::Address as EVMAddress;
+
+const RPC_URL: &str = "http://0.0.0.0:18443";
+const RPC_USERNAME: &str = "rpcuser";
+const RPC_PASSWORD: &str = "rpcpassword";
+const NETWORK: BitcoinNetwork = BitcoinNetwork::Regtest;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    tracing_subscriber::fmt()
+        .with_max_level(tracing::Level::INFO)
+        .init();
+
+    let depositor_private_key = "cVZduZu265sWeAqFYygoDEE1FZ7wV9rpW5qdqjRkUehjaUMWLT1R".to_string();
+
+    let bridge_p2wpkh_mpc_address = "bcrt1qdrzjq2mwlhrnhan94em5sl032zd95m73ud8ddw"
+        .parse::<BitcoinAddress<NetworkUnchecked>>()?
+        .require_network(NETWORK)?;
+
+    let mut inscriber = Inscriber::new(
+        RPC_URL,
+        NETWORK,
+        NodeAuth::UserPass(RPC_USERNAME.to_string(), RPC_PASSWORD.to_string()),
+        &depositor_private_key,
+        None,
+    )
+    .await
+    .context("Failed to create Depositor Inscriber")?;
+
+    let args: Vec<String> = env::args().collect();
+    let amount = args[1].parse::<f64>()?;
+    info!("Depositing {} BTC to L2", amount);
+
+    info!(
+        "Depositor L1 balance: {}",
+        inscriber
+            .get_balance()
+            .await
+            .context("Failed to get balance")?
+    );
+
+    let receiver_l2_address = EVMAddress::random();
```

Comment:
don't we want to deposit to some address? why is it random?

Reply:
Good one, I added one of the EVM rich wallet addresses with the known private key